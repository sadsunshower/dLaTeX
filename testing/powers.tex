\documentclass{article}

\begin{document}

Test: powers

It should catch these: $e^-x$

\[
    3^-x
\]

But not these: $e^{-x}$

\[
    3^{-x}
\]

\end{document}