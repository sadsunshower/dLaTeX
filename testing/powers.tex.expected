This is dLaTeX version 0.1
{"messages": [{"severity": "warning", "message": "Remember to add curly braces {} around powers with more than one symbol.", "line": "It should catch these: $e^-x$", "num": 6, "marker": [25]}, {"severity": "warning", "message": "Remember to add curly braces {} around powers with more than one symbol.", "line": "    3^-x", "num": 9, "marker": [5]}], "error": false}
